packages/shared-models/src/ymodels.ts

```diff
@@ -1031,6 +1050,26 @@ export class YMarkdownCell
       attachments: this.getAttachments()
     };
   }
+
+  /**
+   * Gets whether or not the cell is rendered.
+   *
+   * @returns The cell "rendered" state.
```

Comment:
Getter/setter are not documented with `@returns` and `@params`.  Just one doc string describing the attribute on the getter.

```suggestion
   * Whether or not the cell is rendered.
```